### PR TITLE
for the Mitra auth library enhancement landing into the intro page after login

### PR DESCRIPTION
**Pull request by farhanp1502**

### CODE REVIEW

**Review by 5Amogh — APPROVED** [review]

File: frontend-auth-module/projects/sl-auth-app/src/assets/config/library-config.json [hunk 1]
```diff
@@ -11,5 +11,7 @@
     "otpGenerationApiPathForResetPassword": "/elevate-user/v1/account/generateOtp",
     "supportEmail":"mentoredtest1@yopmail.com",
     "privacyPolicyUrl":"https://www.privacyploicy.com",
-    "termOfUseUrl":"https://www.termsofuse.com"
+    "termOfUseUrl":"https://www.termsofuse.com",
+    "intialPage": true,
```
Comment (Vinod-V3) [inline frontend-auth-module/projects/sl-auth-app/src/assets/config/library-config.json]: Check the spelling of initial in line 15 and 16

File: frontend-auth-module/projects/sl-auth-lib/src/lib/services/auth.service.ts [hunk 2]
```diff
@@ -73,7 +73,12 @@ export class AuthService {
               if (res?.responseCode === "OK") {
                 localStorage.clear();
                 this.sendMessage();
-                this.router.navigate(['/login']);
+                if(this.configData?.intialPage){
```
Comment (Vinod-V3) [inline frontend-auth-module/projects/sl-auth-lib/src/lib/services/auth.service.ts]: spelling check required based on the config change

File: frontend-auth-module/projects/sl-auth-app/src/assets/config/library-config.json (hunk 1 above)
Comment (farhanp1502) [reply frontend-auth-module/projects/sl-auth-app/src/assets/config/library-config.json]: done

File: frontend-auth-module/projects/sl-auth-lib/src/lib/services/auth.service.ts (hunk 2 above)
Comment (farhanp1502) [reply frontend-auth-module/projects/sl-auth-lib/src/lib/services/auth.service.ts]: done

**Pull request merged**